src/test/java/Problem2Test.java

```diff
@@ -8,7 +8,11 @@
     // Okay to change
     @Test
     public void testBubbleSort() {
-        assertTrue(false);
         // add your tests here
+        int sort[] = {6, 4, 5, 3, 2, 1};
+        int answer[] = {1, 2, 3, 4, 5, 6};
+        Problem2 test = new Problem2();
+        test.bubbleSort(sort);
```

Comment:
no need to create a Problem2 object. bubbleSort is a static function. Problem2.bubbleSort should just work. I'm actually surprised this line even compiles.